api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/PatientTranslatorImpl.java

```diff
@@ -79,8 +82,26 @@ public Patient toFhirResource(@Nonnull org.openmrs.Patient openmrsPatient) {
 		
 		Patient patient = new Patient();
 		patient.setId(openmrsPatient.getUuid());
-		patient.setBirthDate(openmrsPatient.getBirthdate());
 		patient.setActive(!openmrsPatient.getVoided());
+		patient.setBirthDate(openmrsPatient.getBirthdate());
+		
+		if (openmrsPatient.getBirthdateEstimated() != null) {
```

Comment:
What happens if the birthdate is not estimated?

Reply:
it will set the birthday as usual. it will alter only if birthday is estimated.

Reply:
I have put all this in if else statement. it will check whether the birthday is estimated or not and according to this we will set the birthday.